src/Microsoft.Sbom.Api/Utils/ComponentDetectorCachedExecutor.cs

```diff
@@ -18,6 +19,7 @@ public class ComponentDetectorCachedExecutor
         private readonly ILogger log;
         private readonly ComponentDetector detector;
         private ConcurrentDictionary<int, ScanResult> results;
+        private readonly TimeSpan componentDetectionTimeout = TimeSpan.FromMinutes(6);
```

Comment:
Should we consider this to be overridable using a parameter or env var?

Reply:
If not, let's bump the timeout to make sure we allocate enough time for large codebases

Reply:
I would prefer making this configurable as well.  Next time let's make that an explicit requirement in the task ;-)